### ISSUE TITLE
How do I add a Column or row in xlwings on Mac

### ISSUE THREAD

**Issue by Speccles96** [xlwings/xlwings#1284]
#### I have found several threads on GitHub asking about this issue of inserting rows and columns. However, all of the sample code people have said works I can not get to work. I am on a Mac and it doesn't seem to work no matter what I try, oddly enough though if I replace insert with delete in my sample code below it will delete a column no problem

```python
599                         selectortype, code = self.AS_appdata.referencebyname()[name]
    600                 except KeyError as e:
--> 601                         raise AttributeError("Unknown property, element or command: {!r}".format(name)) from e
    602                 if selectortype == kProperty:
    603                         return Reference(self.AS_appdata, self.AS_aemreference.property(code))

AttributeError: Unknown property, element or command: 'insert'


```

#### Include a minimal code sample to reproduce the issue (and attach a sample workbook if required!)

```python
wb = xw.Book().sheets[0]
wb.range("A1").api.insert()

```

**Comment by fzumstein** [comment]
this has been implemented since 0.16.4: https://docs.xlwings.org/en/stable/whatsnew.html#v0-16-4-dec-17-2019

```
wb.sheets[0].range('A:A').insert()
```

**Comment by Speccles96** [comment]
> this has been implemented since 0.16.4: https://docs.xlwings.org/en/stable/whatsnew.html#v0-16-4-dec-17-2019
> 
> ```
> wb.sheets[0].range('A:A').insert()
> ```

Yes that works on windows but it does not work on when running xlwings on a Mac

**Comment by fzumstein** [comment]
what error are you seeing? and what's your xlwings version? definitely works over here.
